packages/model/src/user.ts

```diff
@@ -115,6 +115,7 @@ export interface UserPatchRequest {
   skills?: string[];
   skillsDescription?: string;
   questions?: string[];
+  teams?: Omit<UserTeam, 'displayName' | 'role' | 'proposal'>[];
```

Comment:
Shouldn't this be a Pick to match the Joi Schema?

Reply:
Was having a TS error using Pick, I will paste it here

Reply:
`teams?: Pick<UserTeam, 'id' | 'approach' | 'responsibilities'>[];`
seemed to work